components/admin_console/permission_schemes_settings/permissions_tree/permissions_tree.jsx

```diff
@@ -147,7 +147,7 @@ export default class PermissionsTree extends React.Component {
         if (scope === 'team_scope' && this.groups[0].id !== 'teams_team_scope') {
             this.groups[0].id = 'teams_team_scope';
         }
-        if (license && license.isLicensed && license.LDAPGroups === 'true' && !postsGroup.permissions.includes(Permissions.USE_GROUP_MENTIONS)) {
+        if (license && license.IsLicensed === 'true' && license.LDAPGroups === 'true' && !postsGroup.permissions.includes(Permissions.USE_GROUP_MENTIONS)) {
```

Comment:
So I recently was told that the optional chaining is ES6 syntax (which I thought was only Typescript syntax prior) so I guess we can switch these to use it.
```suggestion
        if (license?.IsLicensed === 'true' && license?.LDAPGroups === 'true' && !postsGroup.permissions.includes(Permissions.USE_GROUP_MENTIONS)) {
```